lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -161,15 +161,22 @@ sub email_search : Path('/admin/email-search') Args(0) RequireAuth(account_admin
     my ($self, $c) = @_;
 
     my $form = $c->form(form => 'Admin::EmailSearch');
-    my @results;
-    my $searched = 0;
+    my $results;
+    my $stored_email = $c->session->{searched_email};
```

Comment:
While there isn't really a chance of conflict here, making the key slightly more specific would be nice since these are global variables you can access from any action.
```suggestion
    my $stored_email = $c->session->{admin_searched_email};
```